ci/test_java.sh

```diff
@@ -38,7 +38,7 @@ set +e
 
 rapids-logger "Run Java tests"
 pushd java
-mvn test -B -DCUDF_JNI_ARROW_STATIC=OFF -DCUDF_JNI_ENABLE_PROFILING=OFF
+mvn test -B -DCUDF_JNI_ENABLE_PROFILING=OFF
```

Comment:
This is sort of tangentially related. It's a flag that used to exist but no longer does. The reason it came up was that this PR required me to figure out the proper build and runtime pinnings for arrow and in particular it wasn't immediately obvious what the pin should be in the java build environment because I wasn't sure if it was trying to use a static or dynamic lib for arrow in CI.

Reply:
Thanks!